changelog/@unreleased/pr-6419.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Rewrite logCurrentState to safelog to normal service logs rather than custom yml serialization to file.
```

Comment:
Probably obvious, but might want to say that a sanitised form is safelogged, and the full version is logged at unsafe